containers/dotnet-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# [Choice] .NET version: 6.0, 5.0, 3.1
```

Comment:
Assuming that this works with the `-bullseye` variants of the images, you can update this to:
```suggestion
# [Choice] .NET version: 6.0, 5.0, 3.1, 6.0-bullseye, 5.0-bullseye, 3.1-bullseye, 6.0-focal, 5.0-focal, 3.1-focal
```